oz/redis_sessions/middleware.py

```diff
@@ -22,13 +22,22 @@ def _session_key(self):
                 except:
                     pass
 
-            if not session_id:
+            if session_id is None:
```

Comment:
Keep the old if: we used to have a bug relating to empty `session_id` values. I'm fairly sure it shouldn't get to this point in code, but this is a good safeguard.